src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -59,6 +52,18 @@ void initializeFeatures() {
         initializeDrivingVariants();
     }
 
+    /**
+     * Marked final so that tool authors don't override it. Tool authors should override {@link #onTraversalStart} instead.
+     */
+    @Override
+    protected void onStartup() {
```

Comment:
This should be final, so that tool authors don't override it by mistake (as the comment says!)